prosemirror.md

```
@@ -0,0 +1,700 @@
+    getImageBuffer: options.getImageBuffer,
+    metadata: options.metadata,
+    layout: options.layout,
+  })
+  
+  return new Promise((resolve) => {
+    writeDocx(wordDocument, (buffer) => resolve(buffer))
+  })
+}
+```
+
+#### 2. `src/utils/docxNodes.ts` (create)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)